.azure-pipelines/cleanup.yml

```diff
@@ -1,5 +1,11 @@
 steps:
 - script: |
+    set -x
+    # kill daemon process
```

Comment:
Does it cherry pick from master? Looks like missing the logic to check if the file /tmp/azp_daemon_kill_docker_pid exists and the pid exists.

Reply:
It is cherry pick from master.
I didn't 'set -e'. If there is not this file. Just go ahead.